main.js

```diff
@@ -0,0 +1,80 @@
+// eslint-disable-next-line no-unused-vars
+function openNav() {
+  document.getElementById('myNav').style.width = '100%';
+}
+
+// eslint-disable-next-line no-unused-vars
+function closeNav() {
+  document.getElementById('myNav').style.width = '0%';
+}
```

Comment:
- [ ] Kindly remove all `Eslint-disable` statements from your code. Using Eslint helps us to write clean, maintainable and scalable codes and using `Eslint-disable` statements defeats this purpose. Kindly remove it. 

#### Hint; You should remove the inline function from the element in the HTML file. Then you can target the element using _**DOM**_ [methods](https://www.dynamicwebtraining.com.au/blog/javascript-dom-methods) and write your logic. 💻 👍

Reply:
Done